### PR TITLE
USHIFT-3001: Add initial RF ipv6 tests

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

**Comment by pacevedom** [comment]
/hold
Need to fix a routing issue.

File: test/suites/ipv6/singlestack.robot [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    ${match_string}=    Remove String    ${match_string}    "
+    ${response}=    Run With Kubeconfig    oc get svc,ep -l app\=hello-microshift -n ${NAMESPACE}
+    Should Be Equal As Strings    ${match_string}    ${response}    strip_spaces=True
+
+DNS Entry For Route Should Resolve
+    [Documentation]    Resolve hello-microshift route via mDNS from the hypervisor/RF runner.
+    ...    Expects RF runner host has opened port 5353 for libvirt zone.
+
+    ${result}=    Run Process
+    ...    avahi-resolve-host-name ${HOSTNAME}
```
Comment (pmtk) [inline test/suites/ipv6/singlestack.robot:87]: I see it returned `hello-microshift.cluster.local	2001:db8:ca7:ff::1522`. Do think it's possible to have a race condition with another test that would also expose `hello-microshift.cluster.local`?

Should we check if the IP we got is indeed IPv6?

Comment (pacevedom) [reply test/suites/ipv6/singlestack.robot:87]: Good points, did both.

**Comment by pacevedom** [comment]
Periodic tests failing because of the crel upgrade issue described here: https://redhat-internal.slack.com/archives/C0310LGMQQY/p1721797490189859

**Pull request merged**